MH-FiorellaTheme5/source/javascripts/site.js

```diff
@@ -1 +1,13 @@
-// This is where it all goes :)
+$(".slider").slick({
```

Comment:
As a good practice in `js` files we should use single quotes `'` instead of double quotes `"`